### ISSUE TITLE
Web Platform Use Case: Test new platform APIs shims and polyfills using WPT and Saucelabs

### ISSUE THREAD

**Issue by zouhir** [web-platform-tests/wpt#23898]
**The What**
Help client-side polyfills for new Web platform features get tested via WPT and Saucelabs.

**The Why**
Not all browsers implement a feature at the exact same time and developers tend to rely a lot on client-side shims & polyfills until a feature is available in the browser their customers are using.

While polyfills are heavily used and sometimes for an extended period of time, they are in a lot of cases not held to the same testing standards Web platform features are held onto, and developers cannot easily understand what covered and what not, and if there any caveats.

**Existing Runner Capabilities & Bugs**
It's very promising the WPT runner had support for Saucelabs [see CLI documentation](https://web-platform-tests.org/running-tests/command-line-arguments.html#run), however, we ran into multiple issues that prevented us from successfully using WPT runner, and instead we wrote our own selenium-webdriver + saucelabs runner, in other words "reinvented the wheel".

I filed individual issues for each of our blockers:

- [ ] https://github.com/web-platform-tests/wpt/issues/23877
- [x] https://github.com/web-platform-tests/wpt/issues/23875 (fixed, thanks @stephenmcgruer )
- [ ] https://github.com/web-platform-tests/wpt/issues/23895
- [ ] https://github.com/web-platform-tests/wpt/issues/23896

**Final notes & credits**
We'd love to collaborate with the WPT team to get our ScrollTimeline polyfill tested using WPT's harness tests, and help document our work so other teams do the same thing.

Thanks to @stephenmcgruer for guiding me through various issues I hit with WPT and his partnership to help us simplify our custom WPT webdriver / test runner.

**Comment by stephenmcgruer** [comment]
@zouhir - are you still interested in pursuing using Saucelabs here?

**Comment by zouhir** [comment]
yes we are -- we'd really love to delete all these custom server & webdriver we have here: https://github.com/flackr/scroll-timeline/tree/master/test

**Comment by LukeZielinski** [comment]
Periodic ping - are there still plans to work on this in the next few months?

**Comment by stephenmcgruer** [comment]
Unfortunately at this point I doubt we will be able to action this issue from the Ecosystem-Infra side. If the interactions team (cc @flackr ) are interested in this we could probably re-prioritize in either Q1 or Q2 2021, but we've not heard from them on this at all.

We're also quite ham-stringed by not having access to saucelabs - we'd probably need to arrange at least a temporary account for testing if we wanted to get this working.

Of course, PRs are always welcome :)
